app/components/modal/logout/logout.controller.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+    'use strict';
+    angular.module('FoodOrderingApp')
+        .controller('LogoutController', LogoutController);
+    LogoutController.$inject = ['$rootScope', '$uibModalInstance', '$sessionStorage', '$state', 'APP_CONSTANT'];
+
+    function LogoutController($rootScope, $uibModalInstance, $sessionStorage, $state, APP_CONSTANT) {
+
+        var vm = this;
+        vm.logoutMsg = APP_CONSTANT.LOGOUT_MSG;
+
+
+        vm.ok = function () {
+            clearSession();
+            $uibModalInstance.close();
+            $state.go('login');
+        };
+
+        vm.cancel = function () {
+            $uibModalInstance.dismiss();
+        };
+
+        function clearSession() {
+            $sessionStorage.emailId = '';
+            $sessionStorage.orderList = [];
+            $sessionStorage.order = [];
+            $sessionStorage.role = '';
+            $sessionStorage.balance = '';
+            $rootScope.$broadcast('clearRole', $sessionStorage.role);
+        }
```

Comment:
This will not clear the session, it will clear the value, we need to remove the key as well, You can do something like this:
$window.sessionStorage.removeItem(key);